server/src/main/java/org/opensearch/cluster/node/DiscoveryNodes.java

```diff
@@ -393,74 +393,77 @@ public DiscoveryNode resolveNode(String node) {
                     if (localNodeId != null) {
                         resolvedNodesIds.add(localNodeId);
                     }
-                } else if (nodeId.equals("_master")) {
-                    String masterNodeId = getMasterNodeId();
-                    if (masterNodeId != null) {
-                        resolvedNodesIds.add(masterNodeId);
-                    }
-                } else if (nodeExists(nodeId)) {
-                    resolvedNodesIds.add(nodeId);
-                } else {
-                    for (DiscoveryNode node : this) {
-                        if ("_all".equals(nodeId)
-                            || Regex.simpleMatch(nodeId, node.getName())
-                            || Regex.simpleMatch(nodeId, node.getHostAddress())
-                            || Regex.simpleMatch(nodeId, node.getHostName())) {
-                            resolvedNodesIds.add(node.getId());
+                } else if (nodeId.equals("_master")
+                    || nodeId.equals(String.format("_%s", DiscoveryNodeRole.CLUSTER_MANAGER_ROLE.roleName()))) {
```

Comment:
Although there are lots of lines changes in this file, most of them are indent change, this is the first actual changes (out of 2):
`|| nodeId.equals(String.format("_%s", DiscoveryNodeRole.CLUSTER_MANAGER_ROLE.roleName()))`